core/src/main/java/org/apache/accumulo/core/fate/user/schema/FateSchema.java

```diff
@@ -44,8 +44,8 @@ public static class TxInfoColumnFamily {
     public static final String STR_NAME = "txinfo";
     public static final Text NAME = new Text(STR_NAME);
 
-    public static final String TX_NAME = "txname";
-    public static final ColumnFQ TX_NAME_COLUMN = new ColumnFQ(NAME, new Text(TX_NAME));
+    public static final String FATE_OP = "fateOp";
+    public static final ColumnFQ TX_NAME_COLUMN = new ColumnFQ(NAME, new Text(FATE_OP));
```

Comment:
this still needs to be changed
```suggestion
    public static final ColumnFQ FATE_OP_COLUMN = new ColumnFQ(NAME, new Text(FATE_OP));
```